src/client/components/Table/Table.stories.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableProps,
+  TableRow,
+} from './Table'
+
+export default {
+  title: 'Components/Table',
+  component: Table,
+  parameters: {
+    backgrounds: {
+      default: 'gray100',
+    },
+  },
+  args: {
+    minWidth: '650px',
+  },
+}
+
+function createData(
+  name: string,
+  calories: number,
+  fat: number,
+  carbs: number,
+  protein: number,
+) {
+  return { name, calories, fat, carbs, protein }
+}
+
+const rows = [
+  createData('Frozen yoghurt', 159, 6.0, 24, 4.0),
+  createData('Ice cream sandwich', 237, 9.0, 37, 4.3),
+  createData('Eclair', 262, 16.0, 24, 6.0),
+  createData('Cupcake', 305, 3.7, 67, 4.3),
+  createData('Gingerbread', 356, 16.0, 49, 3.9),
+]
+
+const Template: Story<TableProps> = (args) => (
+  <Table {...args}>
+    <TableHead>
+      <TableRow>
+        <TableHeaderCell>Dessert (100g serving)</TableHeaderCell>
+        <TableHeaderCell align="right">Calories</TableHeaderCell>
+        <TableHeaderCell align="right">Fat&nbsp;(g)</TableHeaderCell>
+        <TableHeaderCell align="right">Carbs&nbsp;(g)</TableHeaderCell>
+        <TableHeaderCell align="right">Protein&nbsp;(g)</TableHeaderCell>
+      </TableRow>
+    </TableHead>
+    <TableBody>
+      {rows.map((row) => (
+        <TableRow key={row.name}>
+          <TableCell>{row.name}</TableCell>
+          <TableCell align="right">{row.calories}</TableCell>
+          <TableCell align="right">{row.fat}</TableCell>
+          <TableCell align="right">{row.carbs}</TableCell>
+          <TableCell align="right">{row.protein}</TableCell>
+        </TableRow>
+      ))}
+    </TableBody>
+  </Table>
+)
+
+export const Default = Template.bind({})
+Default.args = {}
+
+function createDataCarInsuranceData(
+  name: string,
+  traffic: boolean,
+  half: boolean,
+  full: boolean,
+) {
+  return { name, traffic, half, full }
+}
+
+const comparisonRows = [
+  createDataCarInsuranceData('Personskador', true, true, true),
+  createDataCarInsuranceData('Skador på annans egendom', true, true, true),
+  createDataCarInsuranceData('Stöld och inbrott', false, true, true),
+  createDataCarInsuranceData('Brand', false, true, true),
+  createDataCarInsuranceData('Glasskador', false, true, true),
+  createDataCarInsuranceData('Bärgning', false, true, true),
+  createDataCarInsuranceData('Skador på bilen från olycka', false, false, true),
+]
+
+const premiumRows = [
+  createDataCarInsuranceData('Personskador', false, true, true),
+  createDataCarInsuranceData('Skador på annans egendom', false, true, true),
+]
+
+const ComparisonTemplate: Story<TableProps> = (args) => (
+  <>
+    <Table {...args}>
+      <TableHead>
+        <TableRow>
+          <TableHeaderCell></TableHeaderCell>
+          <TableHeaderCell align="center">Trafik</TableHeaderCell>
+          <TableHeaderCell align="center">Halv</TableHeaderCell>
+          <TableHeaderCell align="center">Hel</TableHeaderCell>
```

Comment:
Would be nice if we could create a data structure for this as well. :) Thinking if we move this to PCMS you could first create the columns (Trafik, Halv, Hel) and then you add rows with a label and a boolean for each column

Reply:
Yeah, I'm thinking that the alternative to using the MUI API design is to do like [Ant](https://ant.design/components/table) and pass in the data structure instead. But I'm thinking that that will be another component that translates whatever we get from giraffe/PCMS into this structure. Makes sense?

Reply:
Ah yes, the API for the component is great, I was thinking more on the datastructure that we would get from PCMS later on, so no need to change anything here. 🙌